pkg/volume/azuredd/attacher.go

```diff
@@ -72,23 +72,23 @@ func (a *azureDiskAttacher) Attach(spec *volume.Spec, nodeName types.NodeName) (
 	lun, err := diskController.GetDiskLun(volumeSource.DiskName, volumeSource.DataDiskURI, nodeName)
 	if err == cloudprovider.InstanceNotFound {
 		// Log error and continue with attach
-		klog.Warningf(
-			"Error checking if volume is already attached to current node (%q). Will continue and try attach anyway. err=%v",
-			nodeName, err)
+		klog.InfoS(
+			"Error checking if volume is already attached to current node. Will continue and try attach anyway",
+			"nodeName", nodeName, "err", err)
```

Comment:
Please apply to other usage of nodeName key
```suggestion
			"node", klog.KRef("", nodeName), "err", err)
```
Node is an Kubernetes object and should be reported the same way as Pod and any other object, for example `klog.KRef(podNamespace, podName)`. 
On the other hand node is an non-namespaced object which means that you should normally pass an empty string as namespace `klog.KRef("", nodeName)`.

Reply:
This has been updated, waiting for your review with pleasure. @serathius